packages/valory/skills/olas_week_abci/behaviours.py

```diff
@@ -710,10 +711,11 @@ def evaluate_summary(
         llm_dialogues = cast(LlmDialogues, self.context.llm_dialogues)
 
         # llm request message
+        year_abbreviation = int(str(year)[2:])
         request_llm_message, llm_dialogue = llm_dialogues.create(
             counterparty=str(LLM_CONNECTION_PUBLIC_ID),
             performative=LlmMessage.Performative.REQUEST,
-            prompt_template=tweet_summarizer_prompt.replace("{tweet_text}", text),
+            prompt_template=tweet_summarizer_prompt.format(tweet_text=text),
```

Comment:
I think there was an issue with using format here, but can't recall it what was it. Let's give it a try with format again.

Reply:
Importing `tweet_summarizer_prompt` and running `tweet_summarizer_prompt.format(tweet_text="test")` works locally.